app/src/main/res/layout/dashboard_tools_spotlight_list_item.xml

```diff
@@ -0,0 +1,100 @@
+                android:layout_height="161dp"
+                android:src="@{banner}"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toTopOf="parent"
+                tools:src="@tools:sample/backgrounds/scenic" />
+
+            <ProgressBar
+                style="@style/Widget.GodTools.Tools.Download.ProgressBar"
+                android:layout_width="match_parent"
+                android:layout_height="10dp"
+                android:progress="@{downloadProgress}"
+                android:visibility="gone"
+                app:layout_constraintBottom_toBottomOf="@id/toolsImage"
+                tools:max="12"
+                tools:progress="5"
+                tools:visibility="visible" />
+
+            <com.google.android.material.imageview.ShapeableImageView
+                style="@style/Widget.GodTools.Tools.Card.Action.Favorite"
+                android:layout_marginTop="11dp"
+                android:layout_marginEnd="11dp"
+                android:onClick="@{() -> tool.isAdded ? callbacks.removeTool(tool, primaryTranslation ?? parallelTranslation) : callbacks.addTool(tool.code)}"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintTop_toTopOf="parent"
+                app:selected="@{tool.isAdded}" />
+
+            <TextView
+                android:id="@+id/toolName"
+                style="@style/TextAppearance.GodTools.Tools.Card.Title"
+                android:layout_width="0dp"
+                android:layout_height="wrap_content"
+                android:layout_marginStart="16dp"
+                android:layout_marginEnd="16dp"
+                android:layout_marginTop="10dp"
+                android:lines="2"
+                android:text="@{tool.name}"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toBottomOf="@id/toolsImage"
+                tools:text="Tool Name" />
+
+            <androidx.appcompat.widget.AppCompatTextView
+                android:id="@+id/parallelLanguageText"
+                style="@style/Widget.GodTools.Tools.Card.ParallelLanguage"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_marginStart="16dp"
+                android:layout_marginBottom="12dp"
+                android:text="@{parallelLanguage.getDisplayName(context)}"
+                app:layout_constraintTop_toBottomOf="@id/toolName"
+                app:layout_constraintBottom_toBottomOf="parent"
+                app:layout_constraintStart_toStartOf="parent"
+                app:visibleIf="@{tool.type.supportsParallelLanguage &amp;&amp; parallelLanguage != null &amp;&amp; !primaryTranslation.languageCode.equals(parallelLanguage.code)}"
+                tools:text="Spanish" />
```

Comment:
I'm checking now, but my guess is the visibility of this view is going to need to be a bit more complex than it currently is

Reply:
![Screen Shot 2022-02-23 at 11 37 21 AM](https://user-images.githubusercontent.com/47818/155364443-2b659102-8c89-4e96-955d-88916dae102f.png)

Reply:
instead of using `visibleIf`, I think you will want to use a combination of `goneIf` and `invisibleIf`.

If the user doesn't have a parallel language selected, there is no reason to include the extra space for a parallel language in this card, if a parallel language is selected it should be visible for tools that have it, and the view will need to be invisible for tools that aren't available in the parallel language.

This is also a great place to write some unit tests around the visibility of the parallel language label to make sure it behaves properly

Reply:
@gyasistory have you addressed this feedback yet?